### ISSUE TITLE
on tablet (w720), no CheckBox visible

### ISSUE THREAD

**Issue by SandnerSoft** [Angads25/android-filepicker#1]
Hello,

on my tablet is no CheckBox visible. 
Please Help

![screenshot_2016-08-01-20-38-10-283](https://cloud.githubusercontent.com/assets/10620782/17304832/e0b4bca0-5827-11e6-99a3-7da19e31313d.jpeg)

**Comment by singhangadin** [comment]
Are you sure that you have selected correct selection_type (for eg, Directories won't be having checkboxes in case of FILE_SELECT type but file would)?

**Comment by SandnerSoft** [comment]
i'am use that:

```
DialogProperties properties = new DialogProperties();
        properties.selection_mode = DialogConfigs.SINGLE_MODE;
        properties.selection_type = DialogConfigs.DIR_SELECT;
        properties.extensions = null;
        properties.root = new File(DialogConfigs.DEFAULT_DIR);
```

on smartphone is good
(sorry for my english)

**Comment by Angads25** [comment]
Can you upload a screenshot? I have tried it with my nexus 10 and it is working perfectly....

![device-2016-08-02-001203](https://cloud.githubusercontent.com/assets/7099405/17304990/ebe5031e-5845-11e6-9c77-89f9f8e21643.png)

**Comment by SandnerSoft** [comment]
The screenshot above is the first post.

I believe I have found the error.
It lies on the theme. Here I test my app in the Dark and Light Theme.
In Light Theme it's okay, but Dark Theme checkbox is not visible.
It is available, only the colors do not match.

Can the colors generally somewhat adaptable to the material Theme?

**Comment by singhangadin** [comment]
The colors of widgets adapt the default theme.... For eg. The dialog will have material colors with AppCompat Activity.

**Comment by singhangadin** [comment]
Can we close the issue, or I can help you with something, I am recently working on styling the dialog....

**Comment by SandnerSoft** [comment]
> I am recently working on styling the dialog....

yes, then we can close.

**Comment by singhangadin** [comment]
The issue was fixed in v1.0.3.
